client/blocks/editor-checkout-modal/index.tsx

```diff
@@ -22,23 +21,36 @@ import './style.scss';
 
 const wpcom = wp.undocumented();
 
-class EditorCheckoutModal extends Component {
-	static propTypes = {
-		site: PropTypes.object,
-		isOpen: PropTypes.bool,
-		onClose: PropTypes.func,
-		cartData: PropTypes.object,
-	};
+interface CartData {
+	products: Array< {
+		product_id: number;
+		product_slug: string;
+	} >;
+}
```

Comment:
Okay so I wonder if there is already an interface written for `CartData` somewhere, not a blocker for this PR, something to look up later.